rrmngmnt/nmcli.py

```diff
@@ -656,23 +647,33 @@ def _get_str_value(value):
             """
             return "yes" if value is True else "no"
 
-        common_options = (
-            "type {type} con-name {con_name}"
-        ).format(type=con_type, con_name=con_name)
+        common_options = f"type {con_type} con-name {con_name}"
 
         if ifname:
-            common_options += " ifname {ifname}".format(ifname=ifname)
+            common_options += f" ifname {ifname}"
         if auto_connect is not None:
-            common_options += " autoconnect {value}".format(
-                value=_get_str_value(value=auto_connect)
+            common_options += (
+                f" autoconnect {_get_str_value(value=auto_connect)}"
             )
         if save is not None:
-            common_options += " save {value}".format(
-                value=_get_str_value(value=save)
-            )
+            common_options += f" save {_get_str_value(value=save)}"
 
         return common_options
 
+    @staticmethod
+    def _type_options_builder(**kwargs):
+        """
+        Builds nmcli command type options.
+
+        Args:
+            *args: positional arguments.
+            **kwargs: keyword arguments.
```

Comment:
**kwargs: keyword arguments. is not the way to write a docstring.
please specify what are the kwargs expected here.